flump/__init__.py

```diff
@@ -81,35 +81,21 @@ def register_flump_view(self, view_class, url):
         # on both anyway.
         url = url.rstrip('/')
 
-        get_many_func = (
-            view_func if HttpMethods.GET_MANY < methods else _meth_not_allowed
```

Comment:
![image](https://cloud.githubusercontent.com/assets/556490/25714204/2431010a-30ef-11e7-9ee5-b227578038c4.png)

`<` was not quite short for issubset, `<=` is.